gating/check/run

```diff
@@ -184,7 +184,10 @@ bash scripts/bootstrap-ansible.sh
 
 _download_dependent_components
 
-
+if [ "${RE_JOB_SCENARIO}" = "ansible_lint" ]; then
+   find ./playbooks -type f -regex '.*\.y[a]?ml' -print0 | xargs -0 ansible-lint --exclude=/etc
+   exit
```

Comment:
You need to exit with the return code of ansible lint or this gate will pass every time regardless

Reply:
Agree, would like to see how the job is behaving first. Will add the return code here in the script, after going through the report and fixing the code.